submissions/iakow/friends-app/js/main.js

```diff
@@ -0,0 +1,232 @@
+"use strict";
+
+disableFilters();
+
+const USERS_AMOUNT = 100;
+
+mountUserList(getPlaceholders());
+
+const ALL_USER_CARDS = [];
+
+fetch(`https://randomuser.me/api/?results=${USERS_AMOUNT}`)
+  .then((response) => response.json())
+  .then((response) => {
+    response.results.forEach((userData) => {
+      ALL_USER_CARDS.push(getUserCard(userData));
+    });
+
+    mountUserList(ALL_USER_CARDS);
+    enableFilters();
+  });
+
+addHandlers();
+
+
+function addHandlers() {
+  const filtersBlock = document.querySelector("#filters-block");
+
+  filtersBlock.addEventListener("input", (e) => {
+    const { id } = e.target;
+
+    if (id === "filter-name") {
+      e.target.value = e.target.value.trim();
+    }
+
+    if (id === "filter-age-min" || id === "filter-age-max") {
+      if (e.target.value === "0") e.target.value = "";
+    }
+
+    applyFilters();
+  });
+
+  filtersBlock.addEventListener("click", (e) => {
```

Comment:
This creates redundant event listeners for inputs.
[Separation of concerns](https://en.wikipedia.org/wiki/Separation_of_concerns)